docs/Configuration.md

```diff
@@ -1,5 +1,42 @@
 # Configuration
 
+## Table of contents
+
+* [Introduction](#Introduction)
+* [DB section](#DB-section)
+  * [engine](#engine)
+  * [user](#user)
+  * [password](#password)
+  * [database_host](#database_host)
+  * [database_name](#database_name)
+  * [polling_interval](#polling_interval)
+* [MYSQL section](#MYSQL-section)
+  * [host](#host)
+  * [port](#port)
+  * [user](#user)
+  * [password](#password)
+  * [database](#database)
+* [POSTGRESQL section](#POSTGRESQL-section)
+  * [host](#host)
+  * [port](#port)
+  * [user](#user)
+  * [password](#password)
+  * [database](#database)
```

Comment:
each anchor should be suffixed
```
  * [host](#host-1)
  * [port](#port-1)
  * [user](#user-2)
  * [password](#password-2)
  * [database](#database-1)
```

Reply:
And again. The model is a little bit clumsy. I fix all this.